src/server/controllers/s3.js

```diff
@@ -1,33 +1,17 @@
 const router = require('express').Router
 const ms = require('ms')
-const S3 = require('aws-sdk/clients/s3')
-
-const defaultConfig = {
-  acl: 'public-read',
-  endpoint: 'https://{service}.{region}.amazonaws.com',
-  conditions: [],
-  getKey: (req, filename) => filename
-}
 
 module.exports = function s3 (config) {
-  config = Object.assign({}, defaultConfig, config)
-
   if (typeof config.acl !== 'string') {
     throw new TypeError('s3: The `acl` option must be a string')
   }
   if (typeof config.getKey !== 'function') {
     throw new TypeError('s3: The `getKey` option must be a function')
   }
 
-  const client = new S3({
-    region: config.region,
-    endpoint: config.endpoint,
-    accessKeyId: config.key,
-    secretAccessKey: config.secret
-  })
-
   return router()
     .get('/params', (req, res, next) => {
+      const client = req.uppy.s3Client
```

Comment:
you can add `// @ts-ignore` above this line to ignore the unwanted error by typescript

Reply:
thanks!